api/src/main/java/org/openmrs/module/ugandaemrreports/library/DataFactory.java

```diff
@@ -1069,21 +1069,21 @@ public CohortDefinition getLastVisitInTheQuarter(Concept question, TimeModifier
 
 
     public CohortDefinition getLostToFollowUp() {
-        LostPatientsCohortDefinition cd = new LostPatientsCohortDefinition();
+        PatientsWhoDidntTurnupForScheduledAppointmentCohortDefinition cd = new PatientsWhoDidntTurnupForScheduledAppointmentCohortDefinition();
         cd.setMinimumDays(90);
         cd.addParameter(new Parameter("startDate", "startDate", Date.class));
         cd.addParameter(new Parameter("endDate", "Ending", Date.class));
         return convert(cd, ObjectUtil.toMap("startDate=startDate,endDate=endDate"));
 
     }
     public CohortDefinition getEverLost() {
```

Comment:
@dbaluku Where is this cohort used? Lost ends at 89 days, from 90 days the client is Lost to Followup

I think this cohort should be everLostToFollowup in that case

Reply:
@ssmusoke  actually you are right I am going to eliminate it so that it is not used anywhere mistakenly and also because  the everLost Cohort is already existing.

Reply:
@dbaluku getLostToFollowup() is 90+ while Lost should be less than or equal to 89 days

Reply:
Yes @ssmusoke I agree with u, the getLostToFollowup() sets minimumdays to be 90 and has no maximum days which is the same with 90+ and the getLostClients() sets minimum and maximum days as 30 and 89 respectively